### ISSUE TITLE
TypeError: Failed to decode variable 'time1_bounds': Vectorized indexing with vectorized or outer indexers is not supported. Please use .vindex and .oindex properties to index the array.

### ISSUE THREAD

**Issue by edrewitz** [Unidata/siphon#787]
Hi all!

I noticed an issue that began over the past couple days with the TDSCatalog as I've used this exact same code for a long time and it worked with no issues but over the past day or so I started getting all these errors out of nowhere. I am trying to access TDS for the RTMA data and I get this error: **TypeError: Failed to decode variable 'time1_bounds': Vectorized indexing with vectorized or outer indexers is not supported. Please use .vindex and .oindex properties to index the array.**

My guess is this has something to do with TDSCatalog (though I could be wrong) because like I said, I've used this code for a while with no issues: 

```
                try:
                    rtma_cat = TDSCatalog('https://thredds.ucar.edu/thredds/catalog/grib/NCEP/RTMA/CONUS_2p5km/RTMA_CONUS_2p5km_'+times[0].strftime('%Y%m%d_%H00')+'.grib2/catalog.xml')
                    rtma_data = rtma_cat.datasets['RTMA_CONUS_2p5km_'+times[0].strftime('%Y%m%d_%H00')+'.grib2'].remote_access(use_xarray=True)
                    rtma_data = rtma_data.metpy.parse_cf().metpy.assign_latitude_longitude()
                    rtma_temp = rtma_data['Temperature_Analysis_height_above_ground'].squeeze()
                    rtma_dwpt = rtma_data['Dewpoint_temperature_Analysis_height_above_ground'].squeeze()
            
                    rtma_rh = mpcalc.relative_humidity_from_dewpoint(rtma_temp, rtma_dwpt)
                    print("Data retrieval for " + times[0].strftime('%m/%d/%Y %H00 UTC') + " is successful")
    
                    time = times[0]
                    
                    return rtma_rh *100, time
                    
                except Exception as e:
            
                    print("Relative Humidity Data is unavailiable for "+times[0].strftime('%m/%d/%Y %H00 UTC')+ "\nWill try to download the most recent dataset from "+times[1].strftime('%m/%d/%Y %H00 UTC'))
                    
                    #try:
                    rtma_cat = TDSCatalog('https://thredds.ucar.edu/thredds/catalog/grib/NCEP/RTMA/CONUS_2p5km/RTMA_CONUS_2p5km_'+times[1].strftime('%Y%m%d_%H00')+'.grib2/catalog.xml')
                    rtma_data = rtma_cat.datasets['RTMA_CONUS_2p5km_'+times[1].strftime('%Y%m%d_%H00')+'.grib2'].remote_access(use_xarray=True)
                    rtma_data = rtma_data.metpy.parse_cf().metpy.assign_latitude_longitude()
                    rtma_temp = rtma_data['Temperature_Analysis_height_above_ground'].squeeze()
                    rtma_dwpt = rtma_data['Dewpoint_temperature_Analysis_height_above_ground'].squeeze()
            
                    rtma_rh = mpcalc.relative_humidity_from_dewpoint(rtma_temp, rtma_dwpt)
                    print("Data retrieval for " + times[1].strftime('%m/%d/%Y %H00 UTC') + " is successful")

                    time = times[1]
                    
                    return rtma_rh *100, time
  ```

Thanks for looking into this matter.

**Comment by dopplershift** [comment]
I'm guessing this is an incompatibility with newer versions of xarray. Did you update your environment recently?

**Comment by edrewitz** [comment]
Yes, I have been playing around with my environments. I did some more testing and I believe you are correct. The last working version of xarray that works using siphon TDSCatalog is version 2023.12.0. All the 2024 versions raises this error when downloading the RTMA data via siphon. Thanks for your help and I hope this information helps you as well.

**Comment by edrewitz** [comment]
Yes, I have been playing around with my environments as of late. It seems like there may be a compatibility issue with the latest form of xarray.

**Comment by dopplershift** [comment]
I'm pretty sure this was fixed by #800. We're working on a new release which will include this fix. I'm going to go ahead and close out but feel free to re-open if it's not fixed on main or in the next release.
